### DISCUSSION TITLE
AI Content generator on new item creation

### DISCUSSION THREAD

**Discussion by andiamine** [JoomCoder-com/JoomCCK#92] [Ideas]
New idea to disccuss, Generate content automatically in fields of type when creating a new item, is it a good idea ?

**Comment by Giorgi625** [comment]
For example if I have type person(like actor singer or so on) and I write in title name of person will it write in text area field biography of that person according to data which it will find on internet? 
Or what possibilities it will have?

**Reply by andiamine** [reply]
Yes maybe something like that, still i don't have good ideas of how to implement it or how should be useful. that's why i need more suggestions and ideas :)

And also implementing AI will attract more users :) to use JoomCCK.

**Comment by Giorgi625** [comment]
yes it is hot topic nowadays, so it definitely will attract new users.

**Comment by andiamine** [comment]
Yep, any deep suggestions of how should be implemented and where will be useful for us.

**Comment by devforcrypto** [comment]
interesting idea
